wpilibjIntegrationTests/src/main/java/edu/wpi/first/wpilibj/can/AbstractCANTest.java

```diff
@@ -35,7 +35,7 @@
   public static final double kLimitSettlingTime = 20.0; // timeout in seconds
   public static final double kStartupTime = 0.50;
   public static final double kEncoderPositionTolerance = .75;
-  public static final double kPotentiometerPositionTolerance = 10.0 / 360.0; // +/-10
+  public static final double kPotentiometerPositionTolerance = 0.2; // +/-10
```

Comment:
10/360 is like 0.02777. Did you intend to make it way higher tolerance? Also the comment is now confusing/wrong. +/-10 degrees right?